kos_sim/eval_model.py

```diff
@@ -0,0 +1,352 @@
+"""Real robot walking deployment script using PyKOS interface."""
+
+import argparse
+import asyncio
+import math
+import time
+from copy import deepcopy
+from typing import Dict, List, Tuple, Union
```

Comment:
this is pretty nit but i prefer using python >= 3.11 and using built-in `dict, list, tuple, |` instead of these... it makes it so that whoever uses your code has to use python 3.11